### DISCUSSION TITLE
개별 플레이어의 중력을 스크립트로 컨트롤할 수 있나요?

### DISCUSSION THREAD

**Discussion by Cha-SeongHun** [naverz/zepeto-studio-kor#575] [5. Q&A]
스폰된 각 캐릭터의 중력을 각각 다르게 적용하여 사용하고 싶은데 방법이 있을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 스폰된 캐릭터의 정의가 멀티플레이 게임에서 접속하는 실제 각 플레이어인가요?
아니면 NPC 개념으로 생성된 캐릭터인가요?

**Reply by Cha-SeongHun** [reply]
멀티플레이 게임에서 접속시 생성하게 되는 ZepetoPlayer를 뜻합니다!

**Reply by Zepeto-TechEvangelist** [reply]
캐릭터 움직임에 중력값을 적용하는 기능은 추가될 예정입니다.
그전에는 jumpPower값으로 원하는 효과를 낼 수 있을 것 같습니다.

ex) 
this.zepetoCharacter.additionalJumpPower = 5;
